desktop/diskStorageManager.js

```diff
@@ -1,46 +1,52 @@
-// @flow
 import os from 'os';
-import drivelist from 'drivelist';
-import { ipcConsts } from '../app/vars';
+import { ipcConsts, localNodeConsts } from '../app/vars';
```

Comment:
localNodeConsts are used only in diskStorageManager and not app wide, move them to manager